src/components/Navbar/index.js

```diff
@@ -28,6 +37,9 @@ export const Navbar = () => {
           <Link to='/theming'>Theming</Link>
         </li>
       </ul>
+      <div onClick={toggleNavbar} className='navbar-toggle'>
```

Comment:
It is considered bad practice to add an onClick event to an element other than a button. This is for semantic html and accessibility issues, maybe you could try something like:

```html
<button onClick={toggleNavbar} className='navbar-toggle'>
    <GiHamburgerMenu size={25} />
</button>
```

```css
.navbar-toggle {
  color: aliceblue;
  background: transparent;
  border: none;
  cursor: pointer;
  padding: 1em;
  top: 0;
  left: 0;
  position: absolute;
  display: fixed;
  transition: 0.5s;
  &:hover {
    background: #04001a;
  }
}
```